.github/workflows/ruby.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         env:
           RAILS_ENV: test
           DATABASE_URL: "postgresql://doubleup:doubleuptestpassword@localhost"
+          SLACK_SIGNING_SECRET: "FICTIONAL63fZabecd7c3c89387b6a9X"
```

Comment:
This signing secret is made up for test. No real secret is committed to this repo.

Reply:
In general I attempt to avoid directly mocking something from a gem. I do like the idea that we can remove the .env dependency for local tests though (they'll fail without a `SLACK_SIGNING_SECRET` env var). If we switch to `Slack::Events.config.signing_secret` (makes sense) is there a reason was can't just set that to a hard coded value for the `test` environment?

Reply:
This doesn't block merge because I don't feel strongly either way. I mentioned it because I've found it really helpful to reduce the implicit dependencies that may exist for a test whenever possible. Environment variables tend to fall into the category of very implicit, so I generally try to minimize where they're referenced.